feature/staticroute/TwoPortConfig/static_route_test.go

```diff
@@ -0,0 +1,101 @@
+package gnmi_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func TestGNMI(t *testing.T) {
+
+	// Configure a DUT
+
+	dut := ondatra.DUT(t, "dut")
+	dp1 := dut.Port(t, "port1")
+	dp2 := dut.Port(t, "port2")
+
+	ifCfg1 := &telemetry.Interface{
```

Comment:
We might want to say something like:

```golang

config := map[string]string{
  "port1": "192.0.2.10",
  "port2": "192.0.2.20",
}

for port, ipaddr := range config {
   // Do the port configuration here.
   ifCfg := ...GetOrCreateInterface(port)....GetOrCreateAddress(ipaddr)
   dut.Config().Interface(...).Update(t, ifcfg)
}
```

Reply:
Or actually... should we look at [`Attributes`](https://github.com/openconfig/featureprofiles/blob/71497a3b4c5d5806e8425f5b7ee748231d5a53ec/internal/attrs/attrs.go#L35)

```golang

config := []*Attributes{{
  IPv4: "192.0.2.20",
  Name: "port1",
}, {
  IPv4: "192.0.2.30",
  Name: "port2",
}}

for _, value := range config {
   ... same configuration with value.Name and value.IPv4 ...
   ... can include the IPv4 prefix length ...
}
```